org.eclipse.xtext.junit5.tests/.classpath

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-11">
+		<attributes>
+			<attribute name="module" value="true"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
+	<classpathentry kind="src" output="target/test-classes" path="src">
```

Comment:
This is not needed, since we only have tests in this project. It also has the drawback that Eclipse will generate .class files into target/test-classes while Tycho will generate it in target/classes because in build.properties "src" is a source folder.
I will update this also in the other projects.